acceptance-tests/aws/postgres_validator/index.js

```diff
@@ -0,0 +1,36 @@
+const restify = require('restify');
+const vcapServices = require('vcap_services');
+const testPostgres = require('./test')
+
+function runServer(content) {
+    const server = restify.createServer();
+    server.get('/', (_, res, next) => {
+        res.send(content)
+        next()
```

Comment:
Is it usual to call `next()` when also sending a response?

Reply:
I'll delete that next() cause we are definitely not doing anything there. However looking at docs you definitely can call it (just for doing extra things that doesn't have to modify the response that was already sent)